gcc/rust/ast/rust-item.h

```diff
@@ -570,7 +586,10 @@ class FunctionParam
   FunctionParam &operator= (FunctionParam &&other) = default;
 
   // Returns whether FunctionParam is in an invalid state.
-  bool is_error () const { return param_name == nullptr || type == nullptr; }
+  bool is_error () const
+  {
+    return (param_name == nullptr || type == nullptr) && !variadic;
```

Comment:
this is hard to read to me... I would have a if (varadic) { return error_condition } else param_name == nullptr || type ==nullptr

Reply:
Fixed!